views/movies/movie-details.hbs

```diff
@@ -0,0 +1,16 @@
+<h2>Movie Details</h2>
+
+
+<h3></h3>{{this.movieDetails.title}}</h3>
+<p>Genre: {{this.movieDetails.genre}}</p>
+<p>Plot: {{this.movieDetails.plot}}</p>
```

Comment:
your variable has a name `movieDetails` , which means you do not need keyword `this` anymore. We use this if variable you send from js to view has no name